src/_guides/language/language-tour.md

```diff
@@ -983,66 +983,36 @@ For more information about maps, see
 [Generics](#generics) and
 [Maps](/guides/libraries/library-tour#maps).
 
-### Runes
+<a id="runes"></a>
+### Runes and grapheme clusters
 
 In Dart, runes are the UTF-32 code points of a string.
+As of Dart 2.6, instead of using runes use the
```

Comment:
Link `runes` to https://api.dartlang.org/stable/2.7.0/dart-core/Runes-class.html ?